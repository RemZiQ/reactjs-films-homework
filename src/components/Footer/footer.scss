.footer {
  height: 225px;
}

.footerLogo {
  min-height: 150px;
  width: 500px;
  margin: 0 auto;
  line-height: 150px;
  text-align: center;
  font-size: 50px;
  letter-spacing: 2px;
  color: #0aaee4;
  font-family: "Helvetica Neue";
  font-weight: 800;
  text-transform: uppercase;
}

.footerCopyright{
  width: 500px;
  margin: 0 auto;
  text-align: center;
  font-size: 24px;
  letter-spacing: 1px;
  line-height: 44px;
  color: #242b2d;
  font-family: "Helvetica Neue";
}