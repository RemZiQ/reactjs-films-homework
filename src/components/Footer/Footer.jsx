import React from 'react';

import './footer.scss';


const Footer = () => (
  <footer className="footer">
    <div className="footerLogo">films</div>
    <div className="footerCopyright">Copyright Â© 2019 FILMS. Yegor Remizov</div>
  </footer>
);

export default Footer;
