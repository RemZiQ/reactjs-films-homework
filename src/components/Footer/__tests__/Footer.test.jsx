import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Footer from '../Footer';

it('Footer renders well', () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<Footer />);
  expect(result).toMatchSnapshot();
});