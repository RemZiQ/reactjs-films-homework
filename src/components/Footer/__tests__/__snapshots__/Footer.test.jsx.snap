// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders well 1`] = `
<footer
  className="footer"
>
  <div
    className="footerLogo"
  >
    films
  </div>
  <div
    className="footerCopyright"
  >
    Copyright Â© 2019 FILMS. Yegor Remizov
  </div>
</footer>
`;
